# 检测结果数组消息
Header header
Detection[] detections
int32 detection_count
string model_name
float32 inference_time